Lumi/src/RecoNPOTMaker_module.cc

```diff
@@ -212,8 +212,8 @@ namespace mu2e {
     }
     else if(_debugLevel > 0) {std:: cout <<"[RecoNPOTMaker::produce] Did not find DeltaFinder IntensityInfoTimeCluster data" << std::endl;}
 
-    std::unique_ptr<ProtonBunchIntensity> recoPBI (new ProtonBunchIntensity);
-    recoPBI->set(POT_caloEnergy); // only using calo energy to make the estimate for now
+    // only using calo energy to make the estimate for now, with a fixed 10% uncertainty
+    std::unique_ptr<RecoProtonBunchIntensity> recoPBI (new RecoProtonBunchIntensity(POT_caloEnergy, 0.1*POT_caloEnergy));
```

Comment:
A cleaner line to do the smae thing:
`    auto recoPBI = std::make_unique<ProtonBunchIntensity>(POT_caloEnergy); `

Reply:
I updated to this style, thank you for the suggestion!